src/sql/src/plan/error.rs

```diff
@@ -125,3 +161,20 @@ impl From<EvalError> for PlanError {
         PlanError::Unstructured(format!("{:#}", e))
     }
 }
+
+struct ColumnDisplay<'a> {
+    table: &'a Option<PartialName>,
+    column: &'a ColumnName,
+}
+
+impl<'a> fmt::Display for ColumnDisplay<'a> {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        f.write_str("\"")?;
+        if let Some(table) = &self.table {
+            write!(f, "{}", table.item)?;
```

Comment:
`item` is a `String` so if it contains a double quote this will not be correctly escaped. We maybe don't care that hard for error messages.